src/index.js

```diff
@@ -0,0 +1,128 @@
+import Todo from './script/todo';
+import Project from './script/project';
+import './style/app.scss';
+
+let todoProject = JSON.parse(window.localStorage.getItem('projects'));
+if (todoProject == null) {
+  todoProject = [];
+}
+
+function updateLocalStorage(array) {
+  window.localStorage.setItem('projects', JSON.stringify(array));
+}
+
+let currentProject = todoProject[todoProject.length - 1];
+
+const addProject = document.getElementById('add-project');
+const projectName = document.getElementById('projectName');
+const projectList = document.getElementById('projectList');
+const submitProject = document.getElementById('submit');
+const createTodo = document.getElementById('create-todo');
+const projectTitle = document.getElementById('project-title');
+const tbody = document.getElementById('tbody');
+const projectForm = document.getElementById('projectForm');
+projectForm.style.display = 'none';
+
+function addProjectForm() {
+  if (projectForm.style.display === 'none') {
+    projectForm.style.display = 'block';
+  } else {
+    projectForm.style.display = 'none';
+  }
+}
+
+function getProject(project = null) {
+  if (project === null) {
+    return currentProject;
+  }
+  return todoProject[project];
+}
+
+function getProjectName() {
+  if (projectName.value !== '') {
+    const project = document.createElement('div');
+    project.setAttribute('data-index', (todoProject.length - 1) + 1);
+    project.textContent = projectName.value;
+    const projectObj = new Project(projectName.value);
+    todoProject.push(projectObj);
+    updateLocalStorage(todoProject);
+    projectList.appendChild(project);
+    currentProject = getProject(parseInt(project.getAttribute('data-index'), 10));
+    projectName.value = '';
+  }
+}
+
+function renderTodo(currProject) {
```

Comment:
Let us move this function out.
![image](https://user-images.githubusercontent.com/31897434/74271596-7325eb80-4d0d-11ea-8c5a-3ffc5a7d0208.png)
https://www.theodinproject.com/courses/javascript/lessons/todo-list